bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -151,10 +161,21 @@
         <comment>rel3</comment>
         <sqlFile path="V1_71__PatientSearchSql.sql"/>
     </changeSet>
+    <changeSet id="bahmni-201906170941" author="Samuel34">
+    	<preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM global_property where property = 'emr.primaryIdentifierType';
+            </sqlCheck>
+        </preConditions>
+        <comment>Introducing 'emr.primaryIdentifierType' GlobalProperty</comment>
+        <sql>
+        	INSERT INTO global_property (property, uuid, description) values ('emr.primaryIdentifierType', UUID(), 'Primary identifier type for looking up patients, generating barcodes, etc');
+        </sql>
+    </changeSet>
     <changeSet id="bahmni-35" author="tw" context="rel3">
-        <preConditions onFail="HALT">
+    	<preConditions onFail="HALT">
             <sqlCheck expectedResult="1">
-                SELECT COUNT(*) FROM global_property where property = 'emr.primaryIdentifierType'
+                SELECT COUNT(*) FROM global_property where property = 'emr.primaryIdentifierType';
```

Comment:
@samuelmale , adding the `;` here does change the `changeSet` checksum. I suggest to drop that change.